lambdas/raster_tiler/lambda_function.py

```diff
@@ -99,20 +99,28 @@ def apply_annual_loss_filter(
         year <= _end_year - 2000
     )
 
-    red: ndarray = np.ones(intensity.shape).astype("uint8") * 228
+    # Construct RGBA bands with floating point values for precision
```

Comment:
Any chance you can describe in a comment the type of intensity.shape, and what the possible range of its values are?  That would also make it clearer why float32 is the right choice for maintaining accuracy (and for instance, why you don't need to use float64).